modules/ROOT/pages/auto-updates.adoc

```diff
@@ -7,6 +7,38 @@ By default, the OS performs continuous auto-updates via two components:
  * https://github.com/coreos/rpm-ostree[rpm-ostree] handles multiple on-disk OSTree deployments, and it can switch between them at boot-time.
  * https://github.com/coreos/zincati[Zincati] continuously checks for OS updates and applies them via rpm-ostree.
 
+== Configuring Update/Rollout Wariness
+
+The Zincati service runs on a Fedora CoreOS node and checks with the remote Cincinnati service to see when updates are available. As part of the request it can send along a configuration value known as `rollout_wariness` (see https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md#phased-rollouts-client-wariness-canaries[documentation]) to let the server know how eager, or how risk-averse the node is to receiving updates. `rollout_wariness` can be a floating point value between `0.0` (most eager) and `1.0` (most conservative). To configure a node to receive an update as soon as it is available you can set the `rollout_wariness` to `0.0` in a configuration file within `/etc/zincati/config.d/` using https://github.com/coreos/fcct[FCCT]:
+
+.Example FCC adding a file to configure rollout_wariness
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa AAA...
+storage:
+  files:
+    - path: /etc/zincati/config.d/51-rollout-wariness.toml
+      contents:
+        inline: |
+          [identity]
+          rollout_wariness = 0.0
+----
+
+The FCC can then be converted to an Ignition config by using FCCT:
```

Comment:
I would drop this and the shell-line below. It's already covered in the getting started and we don't want to duplicate it for every single snippet we show. Longer we should have a single page showing FCCT usage, and possible an inline FCCT->Ignition converter.

Reply:
I dropped the example and linked to the getting started guide